snippets/mt_convert.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python
+
+# simple python file to convert minetest description.txt and depends.txt files to mod.conf files
+# takes one argument for mod folder path
+# license: MIT wsor
+
+import sys
+import os.path
+
+if len(sys.argv) == 1:
+    print("filepath not provided")
+    exit()
+elif not os.path.exists(sys.argv[1]):
+    print("invalid file path")
+    exit()
+elif os.path.exists(os.path.join(sys.argv[1], "mod.conf")):
+    print("mod.conf already exists!")
+    exit()
+
+descpath = os.path.join(sys.argv[1], "description.txt")
+deppath = os.path.join(sys.argv[1], "depends.txt")
+
+conffile = open(os.path.join(sys.argv[1], "mod.conf"), "a")

```

Comment:
Possibly allow running without arguments to run conversion in current directory.

But then also argument against this: it's not that hard to write that dot yourself.

```suggestion
path = (sys.argv[1:2] or ["."])[0]
if not os.path.exists(path):
    print("invalid file path")
    exit()
elif os.path.exists(os.path.join(path, "mod.conf")):
    print("mod.conf already exists!")
    exit()

descpath = os.path.join(path, "description.txt")
deppath = os.path.join(path, "depends.txt")

conffile = open(os.path.join(path, "mod.conf"), "a")
```